### PR TITLE
refactor: 💡 link storybook

### PR DESCRIPTION
**Pull request by ethan-qsh**

### CODE REVIEW

**Comment by sebopensource** [comment]
👋 Thanks for creating a pull request! 🚀 Checkout the storybook we created for it [here](https://sebgroup.github.io/green/pr-344/chlorophyll/).

**Comment by ethan-qsh** [comment]
You can preview the storybook changes here:
https://sebgroup.github.io/green/pr-344/react/?path=/docs/components-link--link

File: libs/react/src/lib/link/link.stories.mdx [hunk 1]
```diff
@@ -17,21 +17,46 @@ export const Template = ({ ...props }) => <Link {...props} />
...
-    args={{
-      children: 'Go to seb.se',
-      href: 'https://seb.se',
-    }}
-  >
-    {Template.bind({})}
-  </Story>
+  <div>
+    <div className="bg-info p-4 my-4">
+      <Link href="Info Link" className="bg-info">
```
Comment (hjalmers) [inline libs/react/src/lib/link/link.stories.mdx:40]:
```suggestion
      <Link href="https://seb.se">
```

File: libs/react/src/lib/link/link.stories.mdx [hunk 2]
```diff
@@ -17,21 +17,46 @@ export const Template = ({ ...props }) => <Link {...props} />
...
+        Go to seb.se
+      </Link>
+    </div>
+    <div className="bg-success p-4 my-4">
+      <Link href="Success Link">Go to seb.se</Link>
```
Comment (hjalmers) [inline libs/react/src/lib/link/link.stories.mdx:45]:
```suggestion
      <Link href="https://seb.se">Go to seb.se</Link>
```

File: libs/react/src/lib/link/link.stories.mdx [hunk 3]
```diff
@@ -17,21 +17,46 @@ export const Template = ({ ...props }) => <Link {...props} />
...
+    </div>
+    <div className="bg-warning p-4 my-4">
+      <Link href="Warning Link">Go to seb.se</Link>
```
Comment (hjalmers) [inline libs/react/src/lib/link/link.stories.mdx:48]:
```suggestion
      <Link href="https://seb.se">Go to seb.se</Link>
```

File: libs/react/src/lib/link/link.stories.mdx [hunk 4]
```diff
@@ -17,21 +17,46 @@ export const Template = ({ ...props }) => <Link {...props} />
...
+    </div>
+    <div className="bg-danger p-4 my-4">
+      <Link href="Danger Link">Go to seb.se</Link>
```
Comment (hjalmers) [inline libs/react/src/lib/link/link.stories.mdx:51]:
```suggestion
      <Link href="https://seb.se">Go to seb.se</Link>
```

**Review by hjalmers — COMMENTED** [review]
Have you added support for target as well?

**Comment by terrance456** [comment]
> Have you added support for target as well?

Yeap, since the props extends to --> `HTMLProps<HTMLAnchorElement>`

**Pull request merged**